package.json

```diff
@@ -77,12 +77,8 @@
     "wait-on": "^6.0.0"
   },
   "dependencies": {
-    "abortcontroller-polyfill": "^1.7.3",
     "browser-tabs-lock": "^1.2.15",
-    "core-js": "^3.23.3",
     "es-cookie": "^1.3.2",
-    "fast-text-encoding": "^1.0.4",
-    "promise-polyfill": "^8.2.3",
     "unfetch": "^4.2.0"
```

Comment:
Can we drop the fetch polyfil too?

Reply:
Good shout, should be able to remove that as well yes.